lychee-lib/src/collector.rs

```diff
@@ -1,223 +1,10 @@
-use std::{
-    collections::HashSet,
-    fmt::Display,
-    path::{Path, PathBuf},
-};
-
-use glob::glob_with;
```

Comment:
Node: All of the removed type definitions below got moved into the the `types` module.